dcap_ql/sys/types/build.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Builds the FFI type bindings for the dcap quote library of the Intel SGX SDK
+
+use bindgen::callbacks::ParseCallbacks;
+
+const DCAP_QL_TYPES: &[&str] = &[
+    "sgx_quote3_error_t",
+    "sgx_ql_path_type_t",
+    "_sgx_ql_request_policy",
+    "_quote3_error_t",
+    "_sgx_ql_qve_collateral_t",
+    "_sgx_ql_qe3_id_t",
+    "_sgx_ql_config_version_t",
+    "_sgx_ql_config_t",
+    "_sgx_ql_qve_collateral_param_t",
```

Comment:
This data type doesn't exist in the version of dcap checked out by the 2.17-reproducible tag on linux-sgx (`dcap_1.11_reproducible`). Is that our relevant DCAP version, or is it 1.14, or the Github master branch?

Reply:
> This data type doesn't exist in the version of dcap checked out by the 2.17-reproducible tag on linux-sgx 
Are you sure the submodule is up to date? `git submodule update --init --recursive`

Looks like it got added in 1.13.  We are currently targeting 1.14
I'm not sure what the `reproducible` tag is about.  The released version is `2.17.100.3` which shows up on the `2.17` tag, https://github.com/intel/linux-sgx/blob/sgx_2.17/common/inc/internal/se_version.h#L34
The `2.17_reproducible` tag shows `2.17.100.2` https://github.com/intel/linux-sgx/blob/sgx_2.17_reproducible/common/inc/internal/se_version.h#L34

We'll need to figure out where we want to document versions for DCAP and the root SGX

Reply:
> We are currently targeting 1.14

Given that version 2.17 of `linux-sgx` (the latest released version) has version 1.11 of dcap as a submodule, I would argue it makes more sense to target that version. Can you fill me in on the reasoning for using 1.14?

Reply:
where are you seeing the 2.17 to 1.11 mapping?

https://github.com/intel/linux-sgx/tree/sgx_2.17/external shows [dcap_source @ 68a77a8](https://github.com/intel/SGXDataCenterAttestationPrimitives/tree/68a77a852cd911a44a97733aec870e9bd93a3b86) which has the following commit message:
> Intel(R) SGX DCAP 1.14 Release

Looking at the `1.14` tag in the DCAP repo, https://github.com/intel/SGXDataCenterAttestationPrimitives/tree/DCAP_1.14, it also shows [68a77a8](https://github.com/intel/SGXDataCenterAttestationPrimitives/commit/68a77a852cd911a44a97733aec870e9bd93a3b86)

Reply:
Oh that is so weird. My local checkout has 1.11, I even did submodule sync and update. Must be some git thing I messed up.

Never mind, ignore me.